src/App/Utilities/BoolToColorConverter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Bit.App.Utilities
+{
+    public class BoolToColorConverter : IValueConverter
```

Comment:
⛏️ I think the name is too general given that inside it's very specific on what this is for. Maybe we could use:
```suggestion
    public class BoolEnablementToTextColorConverter : IValueConverter
```
so it's easy to recognize what's this for without looking at the implementation.